### ISSUE TITLE
Example app should use runtime-only version of vue?

### ISSUE THREAD

**Issue by RavenXce** [rails/webpacker#300]
The hello world example app is currently referencing the full version of vue which includes the compiler. This isn't really needed and isn't the default way of running Vue applications (`import Vue from 'vue'` defaults to the runtime version).

Should the example use this style instead? : https://github.com/vuejs-templates/webpack/issues/215#issuecomment-287652462

**Comment by gauravtiwari** [comment]
@RavenXce Feel free to make a PR please 👍  Looks good to me

**Comment by irrg** [comment]
Possibly related, possibly worth a new issue (possibly worth telling me I'm Doing It Wrong)… I tried to modify the new example to use vue-router. Could never get past `Unknown custom element: <router-view>` etc. 

Considering vue-router (and vuex) are staples…I'd love to see an example that shows what I'm doing wrong.

**Comment by RavenXce** [comment]
That doesn't seem related to this issue. Did you add the component during initialization?

```
import VueRouter from ...

MyComponent new Vue({
   ...,
   components: { VueRouter, ... },
   ...,
});
```

Reference: https://vuejs.org/v2/guide/components.html#Local-Registration

**Comment by irrg** [comment]
I finally got a router to work within the context of webpacker (example: https://github.com/irrg/rails-webpacker-vue-example) but I haven't been able to port my Vue into it yet. I'll figure it out and if needed ask help elsewhere.

That said, the moment I tried to add hot reloading to my linked example, vue-router broke again. So some stuff definitely feels fickle, but it's likely outside of the scope of the webpacker project (and probably firmly in webpack's camp).
